src/jit/optimizer.cpp

```diff
@@ -1119,10 +1119,11 @@ bool Compiler::optExtractInitTestIncr(
 
 /*****************************************************************************
  *
- *  Record the loop in the loop table.
+ *  Record the loop in the loop table.  Return true if successful, false if
+ *  out of entries in loop table.
  */
 
-void Compiler::optRecordLoop(BasicBlock*   head,
+bool Compiler::optRecordLoop(BasicBlock*   head,
```

Comment:
It shouldn't be `lastExit` because that is just something we track while identifying the loop and then throw it away if there were multiple exits.  The code that checks for multiples has a local called `onlyExit` that it populates iff `lastExit` was unique.  So if we want to change the name of this parameter and the comments describing loop structure and the `lpExit` field in the loop table, we could change it to `onlyExit` or perhaps better `uniqueExit`, but that seems unnecessarily verbose to me...